src/coreclr/debug/createdump/threadinfounix.cpp

```diff
@@ -221,6 +223,9 @@ ThreadInfo::GetThreadContext(uint32_t flags, CONTEXT* context) const
         memcpy(context->D, m_vfpRegisters.fpregs, sizeof(context->D));
 #endif
     }
+#elif defined(__loongarch64)
+    assert(!"Unimplemented yet on Loongarch64!");
```

Comment:
You will have to implement this for createdump to function.

Reply:
OK, Thanks~